PodcastGenerator/index.php

```diff
@@ -27,7 +27,14 @@
     }
 }
 
-$episodes = getEpisodes(null, $config);
+$episodes = null;
+// Testing if search
+if(isset($_GET['search']) && $_GET['search'] !== "" && strtolower($_GET['search']) !== "all") {
```

Comment:
Please remove the all exception. Maybe some people call their episodes that way and it is not cross localized. Also it is a potential ddos risk because the pages will then display without pagination (I think)

Reply:
As the index page is used to display the results pagination will be used, thus I have kept the is not blank statement but removed the "all" statement. Please let me know if I should remove the (&& $_GET['search'] !== "") test as well.

Reply:
Also strpos() in searchEpisodes() function does not work with "".

Reply:
Checking for "" is fine but checking for "all" is not IMO. People who use PG in another language that is not English are being discriminated that way.